src/components/_proxied-dot-io/consul/subnav/index.jsx

```diff
@@ -21,7 +21,7 @@ export default function ConsulSubnav({ menuItems }) {
 				{ text: 'Download', url: '/downloads' },
 				{
 					text: 'Try HCP Consul',
-					url: 'https://cloud.hashicorp.com/?utm_source=consul_io&utm_content=top_nav_consul',
+					url: 'https://portal.cloud.hashicorp.com/sign-up?utm_source=docs&utm_content=top_nav_consul',
```

Comment:
🙋are these `utm_source` values correct? My first instinct is that they should have some consistency between them. `docs` seems weird given that we're using `vault_io` for Vault.

Reply:
These are the desired values according to the stakeholder for this work.  These UTMs are in place just to ensure existing behavior on their end continues to work; they are intended to be removed at a later time.